tests/testgpimage.c

```diff
@@ -564,28 +592,28 @@ static void test_getEncoderParameterListSize ()
 	// FIXME: this returns NotImplemented with libgdiplus.
 #if defined(USE_WINDOWS_GDIPLUS)
 	assertEqualInt (status, Ok);
-	assertEqualInt (size, 164);
+	assertEqualInt (size, is_32bit() ? 164 : 184);
 #endif
 
 	status = GdipGetEncoderParameterListSize (image, &gifEncoderClsid, &size);
-// FIXME: this returns FileNotFound with libgdiplus.
+	// FIXME: this returns FileNotFound with libgdiplus.
```

Comment:
bunch of inadvertent indentation changes here and below.

Reply:
They were deliberate as this now aligns with the code below.